### DISCUSSION TITLE
CloudQuery - Infracost integration

### DISCUSSION THREAD

**Discussion by hassankhosseini** [infracost/infracost#1336] [Ideas & Feature Requests]
Hey everyone,

I've been chatting with the good folks at CloudQuery (https://www.cloudquery.io/ - Yevgeny and Ron), and we have been brainstorming ideas on how these two products might be able to integrate to help end users. We had some great discussions, but we thought we should open it up to the community to see if you had any ideas on what you'd like to see!

Comment away, and let's brainstorm.

**Comment by pirateape** [comment]
It would be a great addition.  Would definitely help in some budgeting.
